src/hooks/useFirebase.js

```diff
@@ -21,11 +21,16 @@ export const useFirebase = (collection) => {
     return db.doc(id).delete();
   };
 
+  const newCollection = (token, data) => {
+    return db.doc(token).collection('data').add(data);
```

Comment:
Maybe call the subcollection "items" instead of "data", it could be more descriptive.

Reply:
will do! ☺️

Reply:
done!